pkg/status/templates/remoteconfig.tmpl

```diff
@@ -0,0 +1,14 @@
+{{/*
+NOTE: Changes made to this template should be reflected on the following templates, if applicable:
+* cmd/agent/gui/views/templates/generalStatus.tmpl
+*/}}
+====================
+Remote Configuration
+====================
+    {{ with .remoteConfiguration }}
+    API Key: {{ if .apiKeyScoped }}Authorized{{ else }}Not authorized{{ end }}
+    Feature: {{ if .orgEnabled }}Enabled{{ else }}Disabled{{ end }}
+    {{- if .lastError }}
+    Last error: {{ .lastError }}
+    {{- end }}
```

Comment:
Could we display `Last error: none` instead of nothing ?
Also, is it the last error of the error for the last RC run ? Looking at the code it seems that we reset the field each time we pull data.

Reply:
This is the error of the last RC poll, in case the agent can't contact the backend or gets rejected